ios-11-4-dep-9-4-arm64-hid-sections-lto.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2017-2018, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_11_4_DEP_9_4_ARM64_HID_SECTIONS_LTO_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_11_4_DEP_9_4_ARM64_HID_SECTIONS_LTO_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 11.4)
+set(IOS_DEPLOYMENT_SDK_VERSION 9.4)
```

Comment:
Maximum iOS 9.x version is 9.3 (unfortunately I missed this fact [here](https://github.com/ruslo/polly/pull/257) already).